### ISSUE TITLE
Indent rule fails on array in array

### ISSUE THREAD

**Issue by dai-shi** [dai-shi/es-beautifier#2]
It doesn't fix indent arrays in an array or objects in an array.

Given input:

```
const a = [
[1],
[2],
[3],
];
```

The expected output is:

```
const a = [
  [1],
  [2],
  [3],
];
```

**Comment by dai-shi** [comment]
relevant issue: https://github.com/eslint/eslint/issues/7242

**Comment by dai-shi** [comment]
https://github.com/eslint/eslint/pull/7618

**Comment by dai-shi** [comment]
fixed in v1.0.0.
